server_admin/topics/authentication/webauthn.adoc

```diff
@@ -161,7 +160,7 @@ The configurable items and their description follow.
  For more details, see https://www.w3.org/TR/webauthn/#rp-id[WebAuthn Specification].
 
 |Attestation Conveyance Preference
-|It tells the WebAuthn API implementation on the browser (https://www.w3.org/TR/webauthn/#webauthn-client[WebAuthn Client]) the preference of how to generate an Attestation Statement. However, {project_name} does not verify the Attestation Statement so that only "none" option can be selected. This is an optional configuration item that is applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "none".
+|It tells the WebAuthn API implementation on the browser (https://www.w3.org/TR/webauthn/#webauthn-client[WebAuthn Client]) the preference of how to generate an Attestation Statement. This is an optional configuration item that is applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "none".
```

Comment:
Should be:  registering "a" WebAuthn authenticator.

Reply:
Yes, I'll fix it as you said.